ystia/forge
# Expose Hybrid demo material

# Pull Request description

## Description of the change

Make all material needed to reproduce the Hybrid GCP/OpenStack/Slurm-Singularity demo publicly available

### How to verify it

1. Setup an Alien4Cloud that could access the 3 infrastructures 
2. Deploy Yorc on those infrastructures
4. Setup 3 orchestrators in a4c  
3. Create an app using the `org.ystia.samples.topologies.ApacheGeneratorVisu` topology
4. Select GCP as location and make sure the compute you deploy on as the correct network tags to expose both 9200 and 5601 ports (on our demo infra it is the `hybrid-demo` tag)
5. On `Review & Deploy` in the `Service Management` tab click on `Expose this deployment as a service `. 
6. Deploy this app
7. Go to `Administration > Services  > Your Service` on the `Location(s)` tab and authorize it on the 3 locations
8. When deployed go to `Kibana > management > Index Patterns > logstash-*` and make it the default index by clicking on the star button
9. Deploy ̀org.ystia.samples.topologies.ApacheGeneratorAnonymization` topology on OpenStack if needed you should configure a proxy in inputs to allow logstash to connect to ES. 
10. After deploying this you should start seeing some data in Kibana `Website analytics` dashboard
11. Deploy `org.ystia.samples.topologies.ApacheLogsRefComputeJob` topology on Slurm. Then launch the run workflow. 
12. In Kibana the `Click cost per domain` dashboard should now be populated.
13. You can repeat the 2 previous steps to update those data

That's it you are now able to use the Hybrid demo


### Description for the changelog

* Expose Hybrid demo material ([GH-93](https://github.com/ystia/forge/issues/93))

## Applicable Issues

Fixes #93
